src/components/UsernameEntry/UsernameEntry.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../utils/constants.scss';
+
+input {
+	color: black;
```

Comment:
```$black```

this hasn't failed because black is a valid value, but you should use the vars specified